encoding/v2/bench/benchmark_eigenda_test.go

```diff
@@ -243,9 +245,15 @@ func BenchmarkFrameGeneration(b *testing.B) {
 		CacheDir:       "../../../resources/srs/SRSTables",
 		NumWorker:      uint64(runtime.GOMAXPROCS(0)),
 	}
+	encodingConfig := encoding.Config{
+		NumWorker:                             uint64(runtime.NumCPU()),
```

Comment:
why not `uint64(runtime.GOMAXPROCS(0))`?

Reply:
good question. I started using NumCPU() recently unknowingly. I guess you're right I should be using GOMAXPROCS because that can be changed dynamically, although in practice probably always behaves the same?

Reply:
[b8e4a4a](https://github.com/Layr-Labs/eigenda/pull/2249/commits/b8e4a4a6909eb3dccebff0528a28bfd6ee978712)